yaml/boundaries.yaml

```diff
@@ -293,6 +282,7 @@ filters:
       <<: [*output_properties, *ne_localized_kind_properties]
       kind: *ne_country_boundaries_kind
       kind_detail: '2'
+      dispute_id: { col: 'ne:brk_a3' }
```

Comment:
Repeating... To indicate both the dispute grouping and the unique line segments, concat with `ne_id` with a `_` spacer. The result would be something like `B05_123456789`

Reply:
Switch to using global, please.

Reply:
@nvkelso is there an example somewhere else of doing this kind of concat in a yaml file?  I haven't found one yet.  If there isn't, I have to export both fields and make a transform to combine them and drop them.  Basically, I can't make a global :D